examples/dev-values-transactions.yaml

```diff
@@ -0,0 +1,119 @@
+#
+#  Copyright 2022 DataStax, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+#
+
+enableAntiAffinity: false
+enableTls: true
```

Comment:
why do you need TLS in this example ?
TLS is not needed for transactions, and this adds some noise.

I understand that maybe you wanted to "test" TLS, but I would turn TLS off in the examples file and make the example file as short as possible